OpenSimRT/Common/src/SignalProcessing.cpp

```diff
@@ -144,7 +142,7 @@ LowPassSmoothFilter::filter(const LowPassSmoothFilter::Input& input) {
     }
 
     // check if dt is consistent
-    if (abs(dt - dtPrev) > 1.5e-3) {
+    if (abs(dt - dtPrev) > 1.5e-3) { // NOTE: ensure double precision if throws
```

Comment:
Try to explain that this value works well in cases of 60Hz (or 0.016) which can sometimes due to floating-point errors become 0.017. It is still strange that we have such a large error on the 3rd digit.